packages/components/src/Combobox/components/ComboboxContent/ComboboxContentList/ComboboxContentList.css

```diff
@@ -61,3 +62,15 @@
 .loadingContainer {
   padding: var(--space-base) var(--space-base);
 }
+
+.loadingContainer.hasOptions {
+  display: flex;
+  justify-content: center;
+
+  position: fixed;
```

Comment:
Minor this, I noticed that the spinner renders weirdly with this position fixed here. It seems to be resolved if this is removed (see video)


https://github.com/GetJobber/atlantis/assets/17303106/f660480c-14d9-40bd-a717-4243c0c6fa65

Reply:
Removing the `position: fixed` and `bottom: 0` seems to resolve this

Reply:
@mattander feel free to ignore this. I just saw that :face-palm:

Reply:
Hah okay! Thanks!